src/Controls/src/Core/FontImageSource.cs

```diff
@@ -42,5 +42,15 @@ public double Size
 			get => (double)GetValue(SizeProperty);
 			set => SetValue(SizeProperty, value);
 		}
+
+		public static readonly BindableProperty ScalingEnableProperty =
+			BindableProperty.Create("ScalingEnable", typeof(bool), typeof(FontImageSource), false,
+				propertyChanged: (b, o, n) => ((FontImageSource)b).OnSourceChanged());
+
+		public bool ScalingEnable
```

Comment:
Not sure if this is totally supported since the image will have to be re-generated when the accessibility settings change. I think we have support for when the density/resolution changes, so this is not crazy.

Reply:
Also the name is different to the other properties.

Reply:
Fixed naming 

Right now nothing resizes when you change accessibility settings even the fonts.

I still need to look at WinUI for this one as well because the CanvasText doesn't have an obvious place to enable/disable scaling whereas iOS/Android seem to have obvious hooks